src/desktop/apps/auction/reducers/__tests__/artworkBrowser.test.js

```diff
@@ -153,6 +153,22 @@ describe("auction/actions/artworkBrowser.test.js", () => {
         ])
         notAllArtworksFetched.artworkBrowser.total.should.eql(10)
       })
+
+      it("updates allFetched to true if the current page exceeds 100", () => {
+        const maxPageResponse = auctions(
+          {
+            artworkBrowser: {
+              total: 1,
+              saleArtworks: [],
+              filterParams: { page: 100 },
+            },
+          },
+          {}
+        )
+        const tooHighPage = auctions(maxPageResponse, actions.updatePage(false))
+        const notAllFetched = auctions(tooHighPage, actions.updateAllFetched())
+        notAllFetched.artworkBrowser.allFetched.should.eql(true)
```

Comment:
And this is now set to `true` (instead of `false`), since we've exceeded the max page of 100, even though our total and number of artworks fetched are different.